src/main/kotlin/lotto/domain/LottoResult.kt

```diff
@@ -0,0 +1,51 @@
+package lotto.domain
+
+import lotto.domain.LottoResult.RateResultString.LOSS
+import lotto.domain.LottoResult.RateResultString.WIN
+import kotlin.math.roundToInt
+
+class LottoResult(tickets: List<LottoTicket>, win: LottoTicket, used: Int = 0) {
+    var first: Int = 0
+    var second: Int = 0
+    var third: Int = 0
+    var fourth: Int = 0
+    private var totalCount: Int = used
+
+    /*
+        3 / 4 / 5 / 6
+        수익률 -> 총상금 / (티켓 수 * 1000) -> 소수 2째 자리 까지 노출
+     */
+    private val totalPrize by lazy {
+        (first * FIRST_PRIZE) + (second * SECOND_PRIZE) + (third * THIRD_PRIZE) + (fourth * FOURTH_PRIZE)
+    }
+    val returnRate by lazy {
+        (totalPrize * 100.0 / (totalCount * TICKET_PRICE)).roundToInt() / 100.0
+    }
+
+    val rateResultString: String
+        get() = if (returnRate > 1) WIN.koreanText else LOSS.koreanText
+
+    init {
+        tickets.forEach {
+            when (win.correctNumberCount(it)) {
+                6 -> first++
+                5 -> second++
+                4 -> third++
+                3 -> fourth++
+            }
+        }
+    }
+
+    private enum class RateResultString(val koreanText: String) {
+        WIN("이익"),
+        LOSS("손해"),
+    }
```

Comment:
```suggestion
    private enum class RateResult(val koreanText: String) {
        WIN("이익"),
        LOSS("손해"),
    }
```

RateResult가 더 어울리지 않을까요? 

```
val rateResult: RateResult
        get() = if (returnRate > 1) WIN else LOSS
```

또한 RateResult 객체를 넘겨주고, ResultView에서 해당 객체의 text를 사용할 수는 없으려나요?

Reply:
RateResult 를 감추고 싶었던것 같아요. 그러다 보니 네이밍이 어긋 난것 같네요..
리뷰 주신대로 RateResult 를 사용하도록 변경했어요.